wlib/stl/List.cpp

```diff
@@ -0,0 +1,210 @@
+/**
```

Comment:
Try adding minimal comments to .cpp files but rather full on documentation in .h files

Reply:
I've added in a lot of documentation now